lib/ae_mdw/db/name.ex

```diff
@@ -30,6 +31,11 @@ defmodule AeMdw.Db.Name do
     }
   end
 
+  @spec expire_after(pos_integer()) :: pos_integer()
```

Comment:
Are these two types block heights? If so, we can use AeMdw.Blocks.height(). Otherwise, we should use more descriptive names about what these integers represent